.circleci/config.yml

```
@@ -405,6 +405,14 @@ jobs:
             sed -i -e "s|<PASSWORD>|${ELASTIC_PASSWORD}|g" manifests/ralph-env-secret.yaml
             kubectl apply -f manifests/ralph-env-secret.yaml
             kubectl apply -f manifests/ralph-env-cm.yaml
+            if [[ -z "$CIRCLE_TAG" ]] || ! [[ "$CIRCLE_TAG" =~ ^v[0-9]+\.[0-9]+\.[0-9]+-helm$ ]]; then
```

```suggestion
            if [[ -z "${CIRCLE_TAG+x}" ]] || ! [[ "$CIRCLE_TAG" =~ ^v[0-9]+\.[0-9]+\.[0-9]+-helm$ ]]; then
```

see https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash/13864829#13864829

---

Oh didn't know, thanks!